docs/SFTPCopy-action.md

```diff
@@ -0,0 +1,93 @@
+SFTP Copy
+=========
+
+<a href="https://cdap-users.herokuapp.com/"><img alt="Join CDAP community" src="https://cdap-users.herokuapp.com/badge.svg?t=sftp-actions"/></a>
+[![Build Status](https://travis-ci.org/hydrator/sftp-actions.svg?branch=develop)](https://travis-ci.org/hydrator/sftp-actions) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) <img src="https://cdap-users.herokuapp.com/assets/cdap-action.svg"/>
+
+
+CDAP Action plugin to copy files from the SFTP server to the specified destination.
+
+
+Usage Notes
+-----------
+In a typical enterprise setting, it is common to have SFTP server which allows sharing of the files over network.
+Various individuals and applications can upload files to the SFTP server. Consumers of these files can download
+them from SFTP server for further processing. Once processed, the files on the SFTP server are no longer required
+and can be deleted.
+
+SFTP Copy plugin targets the use case where the files can be copied from SFTP server and stored on the desired
```

Comment:
Don't mention plugin or usecase. Focus on the how users can use this plugin. How the source path can be configured, how users can specify login credentials. Any assumptions made for SSHing, any additional SSH params. Mention how this can be used in conjunction with the SFTP delete.